chatty-clownfish/bangazon_sprint_3
# All path casing matches

## Description of Proposed Changes
Product, ProductType, and Order have been changed to lowercase in urls.py and views.py

## Steps to Test

```sh
git fetch --all
git checkout naming
```

python manage.py runserver

Make sure pathing works


## Impacted Areas in Application
REST API

## Mentions @username
@buffard 
@danielreedcombs 
@johnnathanwood
